indico/modules/rb/views/user/rooms.py

```diff
@@ -156,7 +156,9 @@ class WPRoomBookingRoomStats(WPRoomBookingBase):
     def _getBody(self, params):
         params['period_options'] = [
             ('pastmonth', _('Past month')),
```

Comment:
I wonder if people will understand past month vs previous month... maybe something like "Last 30 days" would be clearer?